package.json

```diff
@@ -36,50 +36,50 @@
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-actions": "^8.5.3",
-    "@storybook/addon-controls": "^8.5.3",
-    "@storybook/addon-toolbars": "^8.5.3",
-    "@storybook/components": "^8.5.3",
-    "@storybook/preview-api": "^8.5.3",
+    "@storybook/addon-actions": "^8.6.2",
+    "@storybook/addon-controls": "^8.6.2",
+    "@storybook/addon-toolbars": "^8.6.2",
+    "@storybook/components": "^8.6.2",
+    "@storybook/preview-api": "^8.6.2",
     "@storybook/react": "^8.6.2",
     "@storybook/react-vite": "^8.6.2",
-    "@storybook/test-runner": "^0.21.0",
+    "@storybook/test-runner": "^0.22.0",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.2.0",
     "@testing-library/user-event": "^14.6.1",
     "@types/history": "^4.7.11",
     "@types/jest": "^29.5.14",
-    "@types/node": "^20.17.16",
+    "@types/node": "^20.17.22",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.12",
     "@types/wicg-file-system-access": "^2023.10.5",
-    "cspell": "^8.17.3",
+    "cspell": "^8.17.5",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.1",
-    "msw": "^2.7.0",
+    "msw": "^2.7.3",
     "msw-storybook-addon": "^2.0.4",
     "playwright": "^1.50.1",
-    "prettier": "^3.4.2",
+    "prettier": "^3.5.2",
     "react-select-event": "^5.5.1",
     "storybook": "^8.6.2",
-    "typescript": "^5.7.3",
+    "typescript": "^5.8.2",
```

Comment:
I haven't looked at it closely yet, but [the type failures](https://github.com/gravitational/teleport/actions/runs/13603633500/job/38032553655?pr=52656#step:7:9) might be related to more granular checks for branches in return expressions in 5.8.

https://devblogs.microsoft.com/typescript/announcing-typescript-5-8/#granular-checks-for-branches-in-return-expressions